zerver/webhooks/slack_incoming/doc.md

```diff
@@ -0,0 +1,9 @@
+Basic support for Slack-compatible webhooks.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Use your new webhook URL any place that you would use a Slack webhook.
```

Comment:
Is it worth adding links to the relevant part of Slack's documentation that makes clear which Slack integrations are a "Slack webhook"?